elegant/clean_timepoint_data.py

```diff
@@ -0,0 +1,147 @@
+import pathlib
+import shutil
+import json
+import datetime
+
+import numpy
+
+from zplib import datafile
+from elegant import load_data
+
+def remove_timepoint_for_position(experiment_root, position, timepoint, dry_run=False):
+    """Removes all memory of a given timepoint for a given position (files, metadata, annotations entries)
+
+    Parameters
```

Comment:
Style guide says colons after `Parameters` block and individual parameter names :)